src/zc/buildout/buildout.py

```diff
@@ -384,7 +384,12 @@ def __init__(self, config_file, cloptions,
 
         download_cache = options.get('download-cache')
         extends_cache = options.get('extends-cache')
+
+        if bool_option(options, 'abi-tag-eggs', 'false'):
+            from zc.buildout.pep425tags import get_abi_tag
+            options['eggs-directory'] += '-' + get_abi_tag()
```

Comment:
Yeah, the ideal solution would be if the eggs themselves included the ABI in the same way that wheels do. But I don't know what it would take to accomplish that. This is here and it works 😄